src/parser/MaintainSentences.cpp

```diff
@@ -238,9 +238,14 @@ std::string CreateTagIndexSentence::toString() const {
     buf += " ON ";
     buf += *tagName_;
     buf += " (";
-    std::string columns;
-    folly::join(", ", this->columns(), columns);
-    buf += columns;
+    std::string fields;
+    std::vector<std::string> fieldDefs;
+    for (const auto& field : this->fields()) {
+        std::string f = field.get_name();
+        if (field.__isset.type_length) f + "(" +field.get_type_length() + ")";
```

Comment:
`fieldDefs.emplace_back(f+...)` ? 

Do you forget to use the string result?

Reply:
> `fieldDefs.emplace_back(f+...)` ?
> 
> Do you forget to use the string result?

Good point. Thanks for reminding. fixed.